test/Desktop/DesktopTests.robot

```diff
@@ -1,18 +1,20 @@
 *** Settings ***
 Documentation   Zoomba Desktop Library Tests. Requires Appium Server running on port 4723.
 Library         ../../src/Zoomba/DesktopLibrary.py
-Test Setup      Start App
+Suite Setup     Start App
+Test Setup      Launch Application
 Test Teardown   Quit Application
-Suite Teardown    Close Application
+Suite Teardown    Close All Applications
 
 *** Variables ***
 ${REMOTE_URL}           http://localhost:4723/wd/hub
 ${APP}                  Microsoft.WindowsCalculator_8wekyb3d8bbwe!App
 
 *** Keywords ***
 Start App
-    ${app}=     Open Application        ${REMOTE_URL}     platformName=Windows    deviceName=Windows   app=${APP}
+    Open Application        ${REMOTE_URL}     platformName=Windows    deviceName=Windows   app=${APP}
     Maximize Window
+    Quit Application
```

Comment:
Since Calculator isn't a single instance application (only allows one window of the program at a time) we have to quit the application after the initial load in order to use 'Launch Application' in the Test Setup.